Services/CheckRightsService/Database/Entities/DbRight.cs

```diff
@@ -11,26 +12,13 @@ public class DbRight
         [Required]
         public string Name { get; set; }
         public string Description { get; set; }
+        public ICollection<DbRightUser> UserIds { get; set; }
     }
 
     public class DbRightConfiguration : IEntityTypeConfiguration<DbRight>
```

Comment:
You can remove this code.

Reply:
Why?

Reply:
How else am I gonna get access to the related data?

Reply:
Through direct DbSet<DbRightUser> UserRights?

Reply:
DbRightConfiguration is empty

Reply:
Fixed